src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/help-visibilityFilter.html

```diff
@@ -1,7 +1,15 @@
 <div>
-    The visibility filter is used to filter which warnings that are used for the conditions.
-    It matches the items under "Visible Warnings" under an analysis in CodeSonar.
-
-    Please use the ID number of the filter you want to use. You can find the ID numbers under
-    this url <b>hub_address</b>/savedsearches.html?filter=3&ssdomain=0&ssearchsgrd=Ay_0D_BTl3BTl3AgBg
+    <p dir="auto">A CodeSonar warning visibility filter that will be used by the CodeSonar plugin to determine the total list of warnings when evaluating analysis condition options.</p>
```

Comment:
A CodeSonar warning visibility filter. The plugin will evaluate your specified **CodeSonar conditions** with respect to this filter.

Reply:
I think the boldface "CodeSonar condition" should be plural: **CodeSonar conditions**...